activestorage/active.py

```diff
@@ -362,13 +355,6 @@ def _from_storage(self, ds, indexer, chunks, out_shape, out_dtype, compressor, f
         # Because we do this, we need to read the dataset b-tree now, not as we go, so
         # it is already in cache. If we remove the thread pool from here, we probably
         # wouldn't need to do it before the first one.
```

Comment:
This comment is irrelevant now?

Reply:
> This comment is irrelevant now?

That's my understanding

Reply:
done in 9aca259ab2410c983801728e0db8567a242d42d2